src/windows-nano/1803/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM mcr.microsoft.com/powershell:6.2.0-nanoserver-1803
+
+ENTRYPOINT C:\Windows\System32\cmd.exe
+
+WORKDIR C:\\Setup
+
+# Get remote files
+ADD "https://www.nuget.org/api/v2/package/python/3.7.3" python-3.7.3-nuget-amd64.zip
+
+USER ContainerAdministrator 
+RUN setx /M PATH "%PATH%;C:\Program Files\PowerShell;C:\Python;C:\python\scripts" \
+ && setx /M PYTHONPATH "C:\Python\Lib;C:\Python\DLLs;" 
+
+RUN pwsh -ExecutionPolicy Unrestricted -c Expand-Archive ".\python-3.7.3-nuget-amd64.zip" -DestinationPath c:\PythonTemp -Force \
+    && mkdir c:\Python \
+    && xcopy /s c:\PythonTemp\tools C:\Python \
+    && rd /s /q c:\PythonTemp \
+    && mkdir c:\Work \
```

Comment:
Is `mkdir c:\Work \` used in this layer?  I am wondering if line 32 is all that is necessary?

Reply:
C:\work is used as a parent directory of where I expect to mount in the volumes for payloads, helix scripts, etc